tests/garage/tf/policies/test_maml_policy.py

```diff
@@ -0,0 +1,40 @@
+import tensorflow as tf
+
+from garage.tf.envs import TfEnv
+from garage.tf.policies.gaussian_mlp_policy_with_model import GaussianMLPPolicyWithModel
+from garage.tf.policies import MamlPolicy
+from tests.fixtures import TfGraphTestCase
+from tests.fixtures.envs.dummy import DummyBoxEnv
+
+
+class TestMamlPolicies(TfGraphTestCase):
+    def test_maml_policy(self):
+        box_env = TfEnv(DummyBoxEnv())
+        with self.sess.as_default():
+            policy = GaussianMLPPolicyWithModel(
```

Comment:
what are some asserts you can actually test this class with? right now all it does is construct something.

note that we can mock out the wrapped policy, and only test the relationship between wrapped_policy and MamlPolicy.

Reply:
Done.